src/ebonite/client/base.py

```diff
@@ -168,16 +168,15 @@ def run_service(self, name: str, image: Image, environment: RuntimeEnvironment =
         if environment is None:
             environment = self.push_environment(RuntimeEnvironment(env_name))
 
-        params = {k: v for k, v in kwargs.items() if k in {'ports_mapping'}}
+        from ebonite.build import run_docker_img, DockerContainer
+        params = DockerContainer(name, **{k: v for k, v in kwargs.items() if k in {'ports_mapping'}})
```

Comment:
actually, at first a had another thought - here you hardcode `ports_mapping`, but you can use a function from `pyjackson.utils` which returns class field names (dont remember it's name)

Reply:
New design eliminated this part of code, please take a look